static/src/javascripts/projects/common/modules/onward/related.js

```diff
@@ -76,12 +76,39 @@ define([
 
                 container.setAttribute('data-component', componentName);
 
-                relatedUrl = popularInTag || '/related/' + config.page.pageId + '.json';
+                if (ab.getParticipations().EssentialReadTest1 &&
+                    (ab.getParticipations().EssentialReadTest1.variant === 'automated' || ab.getParticipations().EssentialReadTest1.variant === 'curated')  &&
+                    ab.testCanBeRun('EssentialReadTest1')
+                ) {
```

Comment:
I know it's there already, but referencing ab participations here seems a bit smelly. I haven't got an alternative right now . I;ll let you know if I come up with one ..